### ISSUE TITLE
"TypeError: Cannot read property 'wrapper' of undefined"

### ISSUE THREAD

**Issue by Azghour-Saad** [vinayakkulkarni/v-pip#243]
Hi, 

I can't solve this issue,  Any help !!

![issue](https://user-images.githubusercontent.com/33766249/104585065-cc0bab80-5663-11eb-9cb7-8e7adf9a44e0.PNG)

Thanks in advance .

**Comment by vinayakkulkarni** [comment]
Can you provide an reproducible example please?

**Comment by Azghour-Saad** [comment]
Hi,

**Here's the usage step i took :**


![step1](https://user-images.githubusercontent.com/33766249/104614096-38000b00-5688-11eb-8060-2509701894b2.PNG)

![step2](https://user-images.githubusercontent.com/33766249/104614117-3fbfaf80-5688-11eb-9bee-e2a5ee251574.PNG)

![step3](https://user-images.githubusercontent.com/33766249/104614133-43533680-5688-11eb-8d5d-3781d617de5f.PNG)

![last](https://user-images.githubusercontent.com/33766249/104614481-94fbc100-5688-11eb-98e5-be409a83e384.PNG)

**Comment by vinayakkulkarni** [comment]
Can you please create a codesandbox.io reproducible link?

Thanks

**Comment by taylorrackley** [comment]
If you're using VueJS 2.x make sure you're using the correct version 1.x of v-pip. I had that same issue before figuring out I had the wrong version. 1.2.1 worked for me.

**Comment by DBCOOPER90** [comment]
What about Vue.Js 3 users ? I have an import issue. If I hover with eslint, it tells me :
Could not find a declaration file for module 'v-pip'. '.../node_modules/v-pip/dist/v-pip.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/v-pip` if it exists or add a new declaration (.d.ts) file containing `declare module 'v-pip';`ts(7016)

I tried running that command but it told me the developer had to update that.
I tried adding this : "typings": "dist/v-pip.cjs.js" to the module's package.json but that didn't fix it.

I am currently running v-pip 2.2.0 on VueJS 3.
